assets/src/edit-story/components/checklist/checks/storyMissingPublisherName.js

```diff
@@ -13,29 +13,57 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+/**
+ * External dependencies
+ */
+import { TranslateWithMarkup, __ } from '@web-stories-wp/i18n';
 /**
  * Internal dependencies
  */
+import { Link, THEME_CONSTANTS } from '@web-stories-wp/design-system';
 import { useConfig } from '../../../app';
 import { ChecklistCard, DefaultFooterText } from '../../checklistCard';
 import { PRIORITY_COPY } from '../constants';
 import { useRegisterCheck } from '../countContext';
 
 const StoryMissingPublisherName = () => {
-  const hasPublisherName = useConfig(
-    ({ metadata }) => metadata.publisher?.name.length > 0
+  const { generalSettingsLink, publisherName } = useConfig(
+    ({ metadata, generalSettingsLink }) => ({
+      publisherName: metadata.publisher?.name,
+      generalSettingsLink,
+    })
   );
+  const hasPublisherName = publisherName.length > 0;
 
-  const { title, footer } = PRIORITY_COPY.storyMissingPublisherName;
+  const { title } = PRIORITY_COPY.storyMissingPublisherName;
 
   useRegisterCheck('StoryMissingPublisherName', !hasPublisherName);
 
   return (
     !hasPublisherName && (
       <ChecklistCard
         title={title}
-        footer={<DefaultFooterText>{footer}</DefaultFooterText>}
+        footer={
+          <DefaultFooterText>
+            <TranslateWithMarkup
+              mapping={{
+                a: (
+                  <Link
+                    href={generalSettingsLink}
+                    rel="noreferrer"
+                    target="_blank"
+                    size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
+                  />
+                ),
+              }}
+            >
+              {__(
+                'Your Site Title is used when your Story appears on Google. You can set your Site Title by going to <a>General Settings</a>.',
```

Comment:
```suggestion
                'Your Site Title is used when your story appears on Google. You can set your Site Title by going to <a>General Settings</a>.',
```

Seems weird to capitalize `story` like this. Could even make the `site title` lowercase too.

Reply:
So, i did that for continuity with our other capitalized `Story` instances?  I remember having it lower case in the help center and then updating to use capitalized. I agree that site title should be lowercase. 

![Screen Shot 2021-07-30 at 8 49 05 AM](https://user-images.githubusercontent.com/10720454/127678719-c644a635-ab21-49fb-b2fe-68e433b0d7c1.png)